docs/docs-content/clusters/cluster-management/cluster-proxy.md

```diff
@@ -0,0 +1,223 @@
+---
+sidebar_label: "Configure Applications to Use Proxy Server"
+title: "Configure Applications to Use Proxy Server"
+description: "Guide to configure configurations in a cluster to use a proxy server."
+hide_table_of_contents: false
+sidebar_position: 201
+tags: ["clusters", "cluster management", "proxy"]
+---
+
+You can configure applications that operate in your Palette clusters to use a proxy server to access the internet
+instead of accessing the internet directly.
+
+Before you are able to configure applications in a cluster to use the proxy server, you must first ensure that the
+cluster itself is configured to use the proxy server. This means that the host Operating System (OS) and the Palette
+agent inside the cluster will use the proxy server for its outbound communications. The steps to do this vary depending
+on your environment.
+
+After the cluster is configured to use the proxy server, you can proceed to configure the applications inside the
+cluster to use the proxy server. You can do this by applying the `spectrocloud.com/connection: proxy` label to the
+specific job, deployment, or daemon set to instruct an application to use the proxy settings of the cluster.
+
+## Prerequisites
+
+- An active proxy server reachable by your cluster.
```

Comment:
I added this tip in the Azure proxy section as we have some limitations on what proxy servers we support due to gRPC.  Maybe we should do something similar here? We have a [page](https://docs.spectrocloud.com/architecture/grps-proxy/) in the Architecture section that dives into more details about this topic.

![CleanShot 2024-07-29 at 10 52 17](https://github.com/user-attachments/assets/ec85b062-325a-47a4-bae3-1dd83c83ef10)